testsuite/features/build_validation/add_custom_repositories/srv_wait_for_custom_reposync.feature

```diff
@@ -5,3 +5,6 @@ Feature: Wait for reposync activity to finish after adding custom channels
 
   Scenario: Wait for running reposyncs to finish after adding custom channels
     When I wait until all spacewalk-repo-sync finished
+
+  Scenario: Verify the reposync went fine after adding custom channels
```

Comment:
please also change the scenario name in the other feater (add "after adding products" there)

Reply:
Resolved in new commit